src/main/kotlin/com/sourcegraph/cody/chat/ui/ChatPanel.kt

```diff
@@ -0,0 +1,70 @@
+package com.sourcegraph.cody.chat.ui
+
+import com.intellij.icons.AllIcons
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.VerticalFlowLayout
+import com.intellij.util.IconUtil
+import com.intellij.util.concurrency.annotations.RequiresEdt
+import com.sourcegraph.cody.PromptPanel
+import com.sourcegraph.cody.agent.protocol.ChatMessage
+import com.sourcegraph.cody.chat.ChatSession
+import com.sourcegraph.cody.context.ui.EnhancedContextPanel
+import com.sourcegraph.cody.ui.ChatScrollPane
+import com.sourcegraph.cody.vscode.CancellationToken
+import java.awt.BorderLayout
+import java.awt.Dimension
+import java.awt.FlowLayout
+import javax.swing.BorderFactory
+import javax.swing.JButton
+import javax.swing.JPanel
+
+class ChatPanel(project: Project, chatSession: ChatSession) :
+    JPanel(VerticalFlowLayout(VerticalFlowLayout.CENTER, 0, 0, true, false)) {
+  private val messagesPanel = MessagesPanel(project)
+  private val chatPanel = ChatScrollPane(messagesPanel)
+  private val promptPanel: PromptPanel = PromptPanel(chatSession)
+  private val contextView: EnhancedContextPanel = EnhancedContextPanel(project)
+
+  private val stopGeneratingButton =
```

Comment:
Yes, we can bring it back to original size, I do not think it will be problematic.
Although I like the wider more I think it's probably unnecessary change ^^